itsy.css

```diff
@@ -0,0 +1,87 @@
+body {
+  margin: 0px;
+}
+header {
+  display: flex;
+  background: url(img/lead.jpg);
+  height: 100vh;
+  flex-direction: column;
+  justify-content: space-between;
+
+}
+
+.header-top {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 30px;
+  font-family: 'Amatic SC', 'Heveltica Neue', Arial, sans-serif;
+  font-size: 30px;
+  background: rgba(255,255,255, 0.8);
+
+}
+
+.header__list {
+  list-style: none;
+  float: left;
```

Comment:
please, try to change that to flexbox. We're trying to learn something new here.

Reply:
And add pink hover effect, as you saw during demo.